src/fe-small-advertisements/package.json

```
@@ -19,7 +19,8 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test"
+    "test": "react-scripts test",
+    "lint": "eslint '*/**/*.{js,ts,tsx}' --config ./.eslintrc.js"
```

same as before

---

+1

---

```suggestion
    "lint": "eslint '*/**/*.{ts,tsx}' --config ./.eslintrc.js"
```
@AdrianValdes check this out